1-JavaScript/Week3/homework/ex6-totalCost.js

```diff
@@ -22,21 +22,30 @@ instead!
 -----------------------------------------------------------------------------*/
 const cartForParty = {
   // TODO complete this object
+  juices: 2.3,
+  chips: 4.9,
+  cookies: 6.25,
+  nuts: 5.16,
+  beers: 3.05,
 };
 
-function calculateTotalPrice(/* TODO parameter(s) go here */) {
-  // TODO replace this comment with your code
+function calculateTotalPrice(obj) {
+  let totalPrice = 0;
+  for (const price of Object.values(obj)) {
```

Comment:
100% correct however, we can simplify it a little bit more:
 ```suggestion
    Object.values(obj).forEach((price) => (totalPrice += price));
```